src/cargo/ops/cargo_package.rs

```diff
@@ -911,12 +911,7 @@ fn run_verify(
     let pkg_fingerprint = hash_all(&dst)?;
     let ws = Workspace::ephemeral(new_pkg, config, None, true)?;
 
-    let rustc_args = if pkg
-        .manifest()
-        .unstable_features()
-        .require(Feature::public_dependency())
-        .is_ok()
-    {
+    let rustc_args = if ws.config().cli_unstable().public_dependency {
         // FIXME: Turn this on at some point in the future
         //Some(vec!["-D exported_private_dependencies".to_string()])
```

Comment:
We need to make sure we're tracking this

Reply:
How about add it to the tracking issue or make a new issue for it?